osm2gtfs/creators/stops_creator.py

```diff
@@ -41,6 +41,19 @@ def add_stops_to_feed(self, feed, data):
             # Add stop to feed
             gtfs_stop_id = self._add_stop_to_feed(stop, feed)
 
+    def remove_unused_stops_from_feed(self, feed):
```

Comment:
Could you maybe add a docstring here and above that explains when this method is called?

Reply:
Done.